### ISSUE TITLE
trouble to continue datacamp

### ISSUE THREAD

**Issue by Jessevg99** [janboone/programming_for_economists#7]
Hello,

I'm an EBE student, my tutorial teacher is Filip Schouwenaars.
I have signed up before the 8th of february, so that I should buy premium for datacamp.
I finished python without any troubles. However, i started today with intermediate python, i finished the first chapter and wanted to continue with the second, but i had no access to continue. They said that i should buy premium to get access.... 

Can you look at it?

Jesse van Goethem, u522886

**Comment by janboone** [comment]
Hi Jesse,

In fact you are a member of the group under your tilburguniversity.edu email address. Can you sign in with this email address and see whether it works?

**Comment by Jessevg99** [comment]
Hi Jan,

I have already signed in with my tilburg university email.
I always receive the emails from datacamp, so I don't understand why I
haven't access to the ''premium'' chapters.

Thanks for the quick response.

Jesse

**Comment by janboone** [comment]
Hi Jesse,

But are you signed in at datacamp with this email address (j.p.m.vangoethem etc.)? Because I cannot see your progress either under this address?

Can you log out and then log in with this email address and try one of the premium chapters?

Jan.

**Comment by Jessevg99** [comment]
Hi Jan,

I know what the problem is..
I made a mistake in my email , i filled in ( j.p.m....@tilburguniveristy (i
en s had to switch...) I had 2 accounts and i did datacamp on the wrong one.

I checked if I could do the chapters for premium members on the right
account and i had access to it. So it is fixed.

Thanks for your effort.

Op di 19 mrt. 2019 om 15:01 schreef jan boone <notifications@github.com>
